.travis.yml

```diff
@@ -36,12 +36,12 @@ jobs:
       install:
         - rvm use 2.4 --install --fuzzy
         - gem update --system
-        - gem install sass jekyll:3.2.1 html-proofer
+        - gem install sass jekyll:3.2.1 html-proofer:3.9.3
       script:
         - sbt makeMicrosite
-        - htmlproofer \
-            --allow_hash_href \
-            --url-swap "https\://github\.com/pureconfig/pureconfig/tree/master:file\://$(pwd)" \
+        - htmlproofer
```

Comment:
Stupid question: why are the backslashes not required? I thought something like `\` or `>` had to be done to concatenate many lines into one string. (YAML generally leaves me confused and uncomfortable and I'm mostly ignorant of its intricacies.)

Reply:
In the [Plain Flow Scalar Style](http://yaml.org/spec/1.2/spec.html#id2788859), in which a string is not surrounded by neither single nor double quotes, the `\` character doesn't have any special meaning. The [Double-Quoted Style](http://yaml.org/spec/1.2/spec.html#id2787109) uses a single `\` to indicate where a line ends, but it is also not necessary if lines are to be trimmed and concatenated with a single space.